manifests/globals.pp

```diff
@@ -2,21 +2,29 @@
 #
 # @param enable_yum
 #   Enable rpm content plugin, including syncing of yum content
+#   Accepts empty string value, the default value when no lookup
+#   is found, to allow Hiera configuration of the value.
```

Comment:
With the type alias, I think these comments can be dropped.

Reply:
Ah yea, thanks for catching that. I got too excited and pushed early.